fhir-search/src/test/java/com/ibm/fhir/search/parameters/ParametersSearchUtilTest.java

```diff
@@ -34,24 +29,12 @@
 public class ParametersSearchUtilTest extends BaseSearchTest {
     public static final boolean DEBUG = false;
 
-    @Override
-    @BeforeClass
-    public void setup() {
-        FHIRConfiguration.setConfigHome("src/test/resources");
-    }
```

Comment:
assuming this is in basesearch?

Reply:
yeah, interstingly the BaseSearchTest one was setting it to the built classes:
```
FHIRConfiguration.setConfigHome("target/test-classes");
```

I think either is fine, but having them both was a bit confusing